collect_app/build.gradle

```diff
@@ -151,10 +151,9 @@ android {
 
         // To ensure that ODK Collect is installable on all devices, we don't use
         // abiFilters to exclude any ABIs; but to keep the APK slim, we include
-        // the Mapbox native library only for armeabi-v7a (which works on all
-        // ARMv7+ 32-bit and 64-bit devices), and omit it for all the other ABIs.
-        exclude 'lib/arm64-v8a/libmapbox-gl.so'
-        exclude 'lib/x86/libmapbox-gl.so'
+        // the Mapbox native library only for 32-bit and 64-bit ARM devices and
+        // omit it for all X86 devices.
+        exclude 'lib/x86/libmapbox-gl.so1'
```

Comment:
So when I build the APK (after commenting out all our excludes for mapbox libs) and analyze it this is what I see for the x86 libs:

<img width="1105" alt="Screenshot 2019-07-05 at 17 14 12" src="https://user-images.githubusercontent.com/556280/60734598-699efe80-9f48-11e9-936b-be60a3a39318.png">

Were you seeing a `libmapbox-gl.so1`? I tried both the `debug` and `odkCollectRelease` build types just in case it was related to that.

Reply:
I don't remember seeing `libmapbox-g1.so1` in any of the APKs. What I did verify was unzipping the latest beta and confirming that we had the armeabi-v7a lib. 
<img width="539" alt="Screen Shot 2019-07-05 at 9 24 43 AM" src="https://user-images.githubusercontent.com/32369/60735055-c24da780-9f06-11e9-85ff-d28c16986c4c.png">

Reply:
Right but the code was previously excluding `lib/x86/libmapbox-gl.so`. If I build an APK from this PR I'm seeing this in libs:

<img width="1124" alt="Screenshot 2019-07-05 at 17 46 06" src="https://user-images.githubusercontent.com/556280/60735901-f8ae1580-9f4c-11e9-8825-b450f0778445.png">

I thought you were seeing an `so1` file in there so you changed the extension to that in the `exclude`. Sorry if I'm totally misunderstanding what's going on here 😕.